docs/kyma/docs/05-03-overrides.md

```diff
@@ -32,39 +29,63 @@ Overrides for the component are composed from two sets: **common** overrides and
 
 Kyma uses common overrides for the installation of all components. ConfigMaps and Secrets marked with the label `installer:overrides`, contain the definition. They require no additional label.
 
-Kyma uses component-specific overrides only for the installation of specific components. ConfigMaps and Secrets marked with both `installer:overrides` and `component: <name>` labels, where `<name>` is the component name, contain the definition. Component-specific overrides have precedence over Common ones in case of conflicting entries.
+Kyma uses component-specific overrides only for the installation of specific components. ConfigMaps and Secrets marked with both `installer:overrides` and `component: {component-name}` labels contain the definition. Component-specific overrides have precedence over common ones in case of conflicting entries.
```

Comment:
this line kind of duplicates https://github.com/kyma-project/kyma/pull/3823/files#diff-7eaa133cde0954f569cbbf7a8d05fa68R36 and is already outdated as one label is missing

Reply:
as above